packages/graphql/pubspec.yaml

```
@@ -20,7 +20,7 @@ dependencies:
   collection: ^1.15.0
   web_socket_channel: '>=2.3.0 <=2.4.0'
   stream_channel: ^2.1.0
-  rxdart: ^0.27.1
+  rxdart: ^0.28.0
```

I would go for 
```suggestion
  rxdart: '>=0.27.1 <0.29.0'
```
As you don't have a reason to drop 0.27.1, but I would love to see this merged anyway.